scripts/ci-smoketest.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Travis CI Release / Smoketest Script
+#
+LAYER0_PATH=$GOPATH/src/github.com/quintilesims/layer0
+LAYER0_PREFIX=smoketest
+set -e
+set -x
+
+# Ignore if master branch is not target
+if [ ! "$TRAVIS_BRANCH" == "master" ]; then
+  echo "[INFO] Skipping Smoketest";
+  exit 0
+fi
+
+# Tag repository
+git tag "$LAYER0_PREFIX"
```

Comment:
1 - yes, if we wanted to have multiple smoketests run at once from different PRs. I don't think dockerhub has a limit on image tags?
2. That's no good, that will be a blocker if/when we wanted to run `make release`  in travis. Perhaps we need to create an issue to see if there's anyway we can make travis keep `.git` data?